---
shipbot_ros/ShuttlecockState state
